InnoClinic.AuthorizationAPI/BLL/Exceptions/UserIsNotLoggedIn.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BLL.Exceptions;
+
+public class UserIsNotLoggedIn : Exception
```

Comment:
Better to name it `UserIsNotLoggedInException` like you did with `ForbiddenAccessException `